src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java

```diff
@@ -545,28 +585,25 @@ private void populateSummaryStatisticsFromResponses(
             String recipientEmail = response.recipient;
 
             // Compute number of responses including user's self response
-            int numOfResponses = numResponses.getOrDefault(recipientEmail, 0) + 1;
-            numResponses.put(recipientEmail, numOfResponses);
+            numResponses.put(recipientEmail, numResponses.getOrDefault(recipientEmail, 0) + 1);
 
             // Compute number of responses excluding user's self response
             numResponsesExcludingSelf.putIfAbsent(recipientEmail, 0);
             boolean isSelfResponse = giverEmail.equalsIgnoreCase(recipientEmail);
+
+            // simplify?
             if (!isSelfResponse) {
-                int numOfResponsesExcludingSelf = numResponsesExcludingSelf.get(recipientEmail) + 1;
-                numResponsesExcludingSelf.put(recipientEmail, numOfResponsesExcludingSelf);
+                numResponsesExcludingSelf.put(recipientEmail, numResponsesExcludingSelf.get(recipientEmail) + 1);
             }
 
             // Compute minimum score received
-            double minScoreReceived = Math.min(answer, min.getOrDefault(recipientEmail, answer));
-            min.put(recipientEmail, minScoreReceived);
+            min.put(recipientEmail, Math.min(answer, min.getOrDefault(recipientEmail, answer)));
 
             // Compute maximum score received
-            double maxScoreReceived = Math.max(answer, max.getOrDefault(recipientEmail, answer));
-            max.put(recipientEmail, maxScoreReceived);
+            max.put(recipientEmail, Math.max(answer, max.getOrDefault(recipientEmail, answer)));
 
             // Compute total score received
-            double totalScore = total.getOrDefault(recipientEmail, 0.0) + answer;
-            total.put(recipientEmail, totalScore);
+            total.put(recipientEmail, total.getOrDefault(recipientEmail, 0.0) + answer);
```

Comment:
What is the advantage of the above changes? I hope it is not "to reduce line length".